athena-federation-sdk/src/test/java/com/amazonaws/athena/connector/lambda/serde/ObjectMapperUtil.java

```diff
@@ -31,17 +39,64 @@
 
 public class ObjectMapperUtil
```

Comment:
We should likely rename this since it is no longer based on ObjectMapper- perhaps SerdeTestUtil.